app/models/job_invocation.rb

```diff
@@ -80,6 +88,14 @@ class Jail < Safemode::Jail
     allow :sub_task_for_host, :template_invocations_hosts
   end
 
+  def self.search_by_targeted_host(key, operator, value)
+    { :conditions => sanitize_sql_for_conditions(["hosts.id = #{value}"]), :joins => :targeted_hosts }
```

Comment:
Could we use a placeholder and let `sanitize_sql_for_conditions` do its thing instead of interpolating strings ourselves?

Reply:
Done.